src/server/api/vault/vaultApi.ts

```diff
@@ -119,23 +119,24 @@ export async function getVaultItem(id: string, type?: "apiKey" | "privateKey") {
     ? process.env.OP_PRIVATE_KEY_VAULT_ID!
     : process.env.OP_API_TOKEN_VAULT_ID!;
 
-  try {
-    const createdItem = await fetch(
-      `${vaultApiUrl}/v1/vaults/${vaultId}/items/${id}`,
-      {
-        headers: {
-          Authorization: `Bearer ${vaultToken}`,
-          "Content-Type": "application/json",
-        },
+  const createdItem = await fetch(
+    `${vaultApiUrl}/v1/vaults/${vaultId}/items/${id}`,
+    {
+      headers: {
+        Authorization: `Bearer ${vaultToken}`,
+        "Content-Type": "application/json",
       },
-    );
+    },
+  );
+  const vaultItem = await createdItem.json();
+  if (vaultItem?.status !== 400) {
     console.log(`🔑 Retrieved API Token from Vault`);
-    return await createdItem.json();
-  } catch (error: any) {
-    const errMsg = `Failed to retrieve API Token from Vault: ${error?.message}`;
+  } else {
```

Comment:
improoved error handling - prevously even if item was not retireved from the vault, we still were showing this console.log